rest-api-spec/src/main/resources/rest-api-spec/api/cat.cluster_manager.json

```diff
@@ -0,0 +1,51 @@
+{
+  "cat.cluster_manager":{
+    "documentation":{
+      "url":"https://opensearch.org/docs/latest/opensearch/rest-api/cat/cat-master/",
+      "description":"Returns information about the cluster manager node."
+    },
+    "stability":"stable",
+    "url":{
+      "paths":[
+        {
+          "path":"/_cat/cluster_manager",
+          "methods":[
+            "GET"
+          ]
+        }
+      ]
+    },
+    "params":{
+      "format":{
+        "type":"string",
+        "description":"a short version of the Accept header, e.g. json, yaml"
+      },
+      "local":{
+        "type":"boolean",
+        "description":"Return local information, do not retrieve the state from master node (default: false)"
```

Comment:
Fix "master" language in description here, and in other places in the API spec

Reply:
Thank you! I was too careless, and copied the whole `cat.master.json` file 😂

Reply:
Updated in commit https://github.com/opensearch-project/OpenSearch/pull/2404/commits/be447368a729f997afb8a7471fbadd5cf6d3e226